src/Core/tests/DeviceTests.Shared/HandlerTests/HandlerTestBasement.cs

```diff
@@ -131,45 +130,23 @@ protected void InitializeViewHandler(IElement element, IElementHandler handler,
 			}
 		}
 
-		protected TCustomHandler CreateHandler<THandler, TCustomHandler>(IElement element, IMauiContext mauiContext)
-			where THandler : IElementHandler, new()
-			where TCustomHandler : THandler, new()
+		protected THandler CreateHandler<THandler>(IElement element, IMauiContext mauiContext = null)
+			where THandler : IElementHandler
 		{
-			if (element.Handler is TCustomHandler t)
+			if (element.Handler is THandler t)
 				return t;
 
 			mauiContext ??= MauiContext;
-			var handler = Activator.CreateInstance<TCustomHandler>();
+			var handler = mauiContext.Handlers.GetHandler(element.GetType());
```

Comment:
Sometimes handlers were created with the activator (OG way) sometimes they are created from the handler mappers in the maui context (new way). This PR will make it _always_ use the mapper.